lib/cloudregister/registerutils.py

```diff
@@ -46,6 +46,7 @@
 REGISTERED_SMT_SERVER_DATA_FILE_NAME = 'currentSMTInfo.obj'
 RMT_AS_SCC_PROXY_MARKER = 'rmt_is_scc_proxy'
 REGISTRY_CREDENTIALS_PATH = '/etc/containers/config.json'
+BASHRC_LOCAL_PATH = '/etc/bash.bashrc.local'
```

Comment:
Is our standard bashrc provided by the distro really sourcing this file ?

Reply:
from the `/etc/bash.bashrc`:

```
PLEASE DO NOT CHANGE /etc/bash.bashrc There are chances that your changes
# will be lost during system upgrades.  Instead use /etc/bash.bashrc.local
# for bash ....
```